src/DynamoCore/Engine/EngineController.cs

```diff
@@ -38,6 +38,11 @@ public class EngineController : LogSourceBase, IAstNodeContainer, IDisposable
         /// </summary>
         public event AstBuiltEventHandler AstBuilt;
 
+        /// <summary>
+        /// This event is fired to reload the dummy nodes.
+        /// </summary>
+        public static event Action ReloadDummyNodes;
```

Comment:
think about this when naming events 

```
Do name events with a verb or a verb phrase.

Do give event names a concept of before and after, using the present and past tense. For example, a close event that is raised before a window is closed would be called Closing and one that is raised after the window is closed would be called Closed.

Do not use Before or After prefixes or suffixes to indicate pre and post events.

Do name event handlers (delegates used as types of events) with the EventHandler suffix.

Do use two parameters named sender and e in event handler signatures.

The sender parameter should be of type Object, and the e parameter should be an instance of or inherit from EventArgs.

Do name event argument classes with the EventArgs suffix.
```

Reply:
why is this static? Is there a good reason?

Reply:
thinking more, I think this should be named to represent the state that is happening, not one single method you are using to the event to call...

so sometihng like `VMLibrariesReset`

Reply:
Makes sense. Changed the event name accordingly and the event is not static now.